lib/stove/community.rb

```diff
@@ -86,9 +86,10 @@ def yank(name)
     #
     def connection
       @connection ||= ChefAPI::Connection.new do |conn|
-        conn.endpoint = ENV['STOVE_ENDPOINT'] || Config.endpoint || DEFAULT_ENDPOINT
-        conn.client   = ENV['STOVE_USERNAME'] || Config.username
-        conn.key      = ENV['STOVE_KEY']      || Config.key
+        conn.endpoint   = ENV['STOVE_ENDPOINT'] || Config.endpoint || DEFAULT_ENDPOINT
+        conn.client     = ENV['STOVE_USERNAME'] || Config.username
+        conn.key        = ENV['STOVE_KEY']      || Config.key
+        conn.ssl_verify = ENV['SSL_VERIFY'].nil? ? Config.ssl_verify : ENV['SSL_VERIFY']
```

Comment:
The envvar here should be `STOVE_SSL_VERIFY` for consistency, and I think you want `||` instead of the ternary operator